src/Command.php

```diff
@@ -469,9 +475,13 @@ public function execute()
                         //
                         while (($out = fgets($pipes[1])) !== false) {
                             $this->_stdOut .= $out;
+                            if($this->streamOutput)
+                                echo $out;
```

Comment:
This assumes that there's always a console (tty) connected. But this may not always be the case. Echoing out here also limits the way this feature can be used. It would be much more flexible if we would accept a stream here (e.g. `STDOUT`, `STDERR`, ...). This would be similar to what we do with `stdIn`. So a better way would probably be, to add `setStdOutStream()` and `setStdErrStream()` and then write to the streams here and below.

As this change is not so trivial and there's already a lot going on here, we need to test that nothing breaks.

I can maybe try to implement it that way - but can't promise yet if I find time soon.

Reply:
OK.